transports/azure-service-bus/topology_description_asbs_[5,).partial.md

```diff
@@ -0,0 +1,167 @@
+The **topic-per-event** topology dedicates one Azure Service Bus topic to each *concrete* event type. This design moves away from the single “bundle” topic and its SQL or Correlation filters, thereby reducing filter overhead and distributing messages more evenly across multiple topics.
+
+In the topic-per-event topology:
+
+1. **Publishers** send an event to a specific topic named after the most concrete event type.
+2. **Subscribers** each create a *subscription* under each topic that matches the event(s) they are interested in.
+3. Because there is no single, central “bundle” topic to hold all messages, each published event flows to its own dedicated topic.
+
+```mermaid
+flowchart LR
+    subgraph Publisher
+    P[Publishes<br/>ConcreteEventA]
+    end
+
+    subgraph Service Bus
+    T1[Topic: ConcreteEventA]
+    T2[Topic: ConcreteEventB]
+    end
+
+    subgraph Subscriber
+    S1[Subscribes to<br/>ConcreteEventA]
+    S2[Subscribes to<br/>ConcreteEventB]
+    end
+
+    P -->|Publish| T1
+    S1 -->|Subscribe| T1
+    S2 -->|Subscribe| T2
+```
+
+This design can dramatically reduce filtering overhead, often boosting performance and scalability. Distributing the messages across multiple topics avoids the single-topic bottleneck and mitigates the risk of hitting per-topic subscription and filter limits.
+
+#### Quotas and limitations
+
+A single Azure Service Bus topic [can hold up to 2,000 subscriptions, and each Premium namespace (with one messaging unit) can have up to 1,000 topics](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quotas).
+
+- Subscriptions per topic: 2,000 (Standard/Premium).
+- Topics per Premium namespace: 1,000 per messaging unit.
+- Topic size: 5 GB quota per topic.
+
+By allocating a separate topic for each concrete event type, the overall system can scale more effectively:
+
+- Each topic is dedicated to one event type, so message consumption is isolated.
+- If any single topic hits its 5 GB quota, only that event type is affected, reducing the blast radius of a partial outage.
+- The maximum limit of 1,000 topics per messaging unit can comfortably support hundreds of event types, especially when factoring that not all event types are high-volume
+
+> [!NOTE]
+> If the system has numerous event types beyond these limits, an architectural review is recommended. Additional messaging units or other partitioning strategies may be required.
+
+#### Subscription rule matching
+
+In this topology, no SQL or Correlation filtering is strictly required on the topic itself because messages in a topic are all of the same event type. Subscriptions can use a default “match-all” rule (`1=1`) or the default catch-all rule on each topic subscription.
```

Comment:
```suggestion
In this topology, no SQL or Correlation filtering is required on the topic itself because messages in a topic are all of the same event type. Subscriptions can use a default “match-all” rule (`1=1`) or the default catch-all rule on each topic subscription.
```